packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx

```diff
@@ -32,7 +32,7 @@ const useStyles = makeStyles(({ typography, spacing }) => ({
     width: 80
   },
   title: {
-    fontSize: typography.fontSize.xxxl,
+    fontSize: typography.fontSize.xxxxl,
```

Comment:
the web client does 4xl for this. I kinda like that more, but not sure if this was intentional

Reply:
I'm on board with `4xl`. Thoughts @dylanjeffers ?

Reply:
I wonder if it makes sense to have a function like `spacing(2)` for font size too

Reply:
i would have loved that, however... numbers as the first value in js :/